Blind75/Blind75-1-Two-Sum/src/main/Main.java

```diff
@@ -0,0 +1,13 @@
+public class Main {
```

Comment:
Add a package, use proper java namespace convention.

Reply:
package added also changed the file name to clearly show what is inside the file instead of just  "Main"